reports/Oregon.md

```diff
@@ -3475,10 +3475,69 @@ id: or-portland-213
 * https://twitter.com/hungrybowtie/status/1288734291334053888
 
 
+### Police charge, tackle, and pepper spray protesters | August 1st
+
+Police gave directions to protesters to walk in a certain direction. Protesters followed directions. Police charged at protestors, shoving many to the ground. One protester on a bicycle is knocked over. The protester lands on top of the bike and has trouble getting up. Police pepper spray protesters as well.
+
+tags: protester, tackle, shove, grab, strike, pepper-spray, spray
+
+id: or-portland-225
+
+**Links**
+
+* https://twitter.com/hungrybowtie/status/1289791877223350273
+* https://twitter.com/hungrybowtie/status/1289792999291985922
+* https://twitter.com/hungrybowtie/status/1289794603244072961
+* https://twitter.com/PDocumentarians/status/1289792025580023808
+* https://twitter.com/45thabsurdist/status/1289792288533577729
```

Comment:
Might want to mention the person shoved to the ground who can't get up.

Reply:
Yeah this is the "person who has trouble getting up" there are several angles of the same thing here

Reply:
Ah I didn't see.